src/main/plugin/iso19139.ca.HNAP/layout/layout-custom-fields.xsl

```diff
@@ -822,4 +822,43 @@
 
   </xsl:template>
 
+  <!-- Readonly hierarchyLevel in flat mode - changing the hierarchyLevel may cause issues with the xsd schema validation. -->
```

Comment:
Why do we need this commented code?

Reply:
This was the old code where we decide to disable it. I left there because I didn't know what direction to move forward. Now we decided to reorganize the codelist, then this code is not needed. So i will delete it